docs/adr/014-rollover-2.0.md

```diff
@@ -0,0 +1,66 @@
+# ADR 014 - Rollover 2.0
+
+```
+Author: @thomaseizinger
+Date: 20/01/2022
+Status: Active
+```
+
+## Summary
+
+Revise the design of the rollover protocol to:
+
+1. Run at fixed points in time.
+2. Not require approval by either party.
+
+## Context
+
+Currently, rollover is a protocol that is initiated by the taker pretty much on the hour every hour.
+The exact conditions for triggering are that we want the timestamp of the price event that is used for attestation to be 24h in the future.
+Because the oracle attests on the price exactly every hour, we can rollover to a new price event on the hour.
+
+Rollover is coupled to paying fees.
+In particular, the time window until the attestation can be considered as the current funding period.
+For every new funding period, fees are charged depending on the current market.
+In a bull market, shorts pay longs.
+In a bear market, longs pay shorts.
+
+To harmonize, how these fees are paid and where they are coming from, we want to revise the rollover protocol as per the design below.
+
+## The new protocol
+
+### 1. We define fixed UTC timestamps for the funding periods.
+   
+The exact timestamps are not decided yet but it will be something along the lines of:
+   - Every day at 0400 UTC.
+   - Every day at 0800 UTC.
+   - Every day at 1200 UTC.
+   - Every day at 1600 UTC.
+   - Every day at 2000 UTC.
+   - Every day at 0000 UTC.
```

Comment:
❓ Is such a policy meant to be defined by makers or is it encoded in the software? If it is part of the software, could it be too restrictive for future makers?

Reply:
The idea is that it is encoded in the software / protocol so that once things are p2p, all nodes simply adhere to this and it is not maker policy.